src/pa02/snakes_simulations.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Håvard Brobakken Eig, Olav vikøren Espenes '
+__email__ = 'havardei@nmbu.no, olaves@nmbu.no'
+
+import random
+
+
+class Board:
+    """
+    Defining the board with its snakes, ladders and the goal.
+    """
+    def __init__(self, *args, **kwargs):
+        self.extra = args, kwargs
+        self.ladders = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85}
+        self.snakes = {25: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12, 87: 70}
+        self.resilient_move = False
+        self.lazy_move = False
+        self.goal = 90
+
+    def goal_reached(self, position):
+        """
+        Returns true if a player has reached or moved beyond the goal.
+        """
+        if position >= self.goal:
+            return True
+        return False
```

Comment:
Please have the habit of returning comparison when possible.
`return position >=self.goal`